src/style.css

```
@@ -1 +1,32 @@
-/* add styles here */
+.page {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

you can use `not` and remove margin-right from `stars__star`
```suggestion
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```